src/test/TestGhoStewards.t.sol

```diff
@@ -40,45 +65,231 @@ contract TestGhoStewards is Test {
+    GHO_GSM_STEWARD = new GhoGsmSteward(address(strategyFactory), RISK_COUNCIL);
+    Gsm(GHO_GSM_USDC).grantRole(Gsm(GHO_GSM_USDC).CONFIGURATOR_ROLE(), address(GHO_GSM_STEWARD));
+    Gsm(GHO_GSM_USDT).grantRole(Gsm(GHO_GSM_USDT).CONFIGURATOR_ROLE(), address(GHO_GSM_STEWARD));
+
+    address[] memory controlledFacilitators = new address[](3);
+    controlledFacilitators[0] = address(GHO_ATOKEN);
+    controlledFacilitators[1] = address(GHO_GSM_USDC);
+    controlledFacilitators[2] = address(GHO_GSM_USDT);
+    changePrank(OWNER);
+    GHO_BUCKET_STEWARD.setControlledFacilitator(controlledFacilitators, true);
 
-    // TODO: Find which contracts are already deployed that we need
-    // TODO: Deploy stewards, using corresponding contracts
-    // TODO: Ensure we grant all appropriate roles
+    vm.stopPrank();
   }
 
   function testSetup() public {
     assertEq(
       IAccessControl(ACL_MANAGER).hasRole(
         IACLManager(ACL_MANAGER).RISK_ADMIN_ROLE(),
-        address(ghoAaveSteward)
+        address(GHO_AAVE_STEWARD)
+      ),
+      true
+    );
+
+    assertEq(
+      IAccessControl(GHO_TOKEN).hasRole(
+        GhoToken(GHO_TOKEN).BUCKET_MANAGER_ROLE(),
+        address(GHO_BUCKET_STEWARD)
       ),
       true
     );
+
+    assertEq(
+      IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL).getRateLimitAdmin(),
+      address(GHO_CCIP_STEWARD)
+    );
+    assertEq(
+      IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL).getBridgeLimitAdmin(),
+      address(GHO_CCIP_STEWARD)
+    );
+
+    assertEq(
+      Gsm(GHO_GSM_USDC).hasRole(Gsm(GHO_GSM_USDC).CONFIGURATOR_ROLE(), address(GHO_GSM_STEWARD)),
+      true
+    );
+    assertEq(
+      Gsm(GHO_GSM_USDT).hasRole(Gsm(GHO_GSM_USDT).CONFIGURATOR_ROLE(), address(GHO_GSM_STEWARD)),
+      true
+    );
+  }
+
+  function testGhoAaveStewardUpdateGhoBorrowCap() public {
+    uint256 currentBorrowCap = _getGhoBorrowCap();
+    uint256 newBorrowCap = currentBorrowCap + 1;
+    vm.prank(RISK_COUNCIL);
+    GHO_AAVE_STEWARD.updateGhoBorrowCap(newBorrowCap);
+    assertEq(_getGhoBorrowCap(), newBorrowCap);
+  }
+
+  function testGhoAaveStewardUpdateGhoSupplyCap() public {
+    uint256 currentSupplyCap = _getGhoSupplyCap();
+    assertEq(currentSupplyCap, 0);
+    uint256 newSupplyCap = currentSupplyCap + 1;
+    // Can't update supply cap even by 1 since it's 0, and 100% of 0 is 0
+    vm.expectRevert('INVALID_SUPPLY_CAP_UPDATE');
+    vm.prank(RISK_COUNCIL);
+    GHO_AAVE_STEWARD.updateGhoSupplyCap(newSupplyCap);
+  }
+
+  function testGhoAaveStewardUpdateGhoBorrowRate() public {
+    IDefaultInterestRateStrategyV2.InterestRateData memory currentRates = _getGhoBorrowRates();
+    vm.prank(RISK_COUNCIL);
+    GHO_AAVE_STEWARD.updateGhoBorrowRate(
+      currentRates.optimalUsageRatio,
+      currentRates.baseVariableBorrowRate + 1,
+      currentRates.variableRateSlope1,
+      currentRates.variableRateSlope2
+    );
+    assertEq(_getGhoBorrowRate(), currentRates.baseVariableBorrowRate + 1);
```

Comment:
check other params as well please. Can be done in the same test

Reply:
Done